docs/tutorials/getting_started/command_line.rst

```diff
@@ -38,9 +37,9 @@ Here are some of the plots that are output when running with the latest total nu
 
 The results from the tournament itself (ordered by median score):
 
-.. image:: http://axelrod-python.github.io/tournament/assets/strategies_boxplot.svg
-   :width: 50%
-   :align: center
```

Comment:
This was a decision made a while ago now (originally the tournament was part of the repository), it's because the library has grown so much and become an independent tool for game theoretic research in it's own right. So the tournament is just one of an infinite amount of tournaments that can be run with it. 

It makes sense to have it as a separate 'showcase' repository which helps define the library as a tool and the tournament as one possible output of it. 

Does that make sense?